codegenerator/cli/templates/static/codegen/src/eventFetching/chainWorkers/HyperSyncWorker.res

```diff
@@ -453,6 +435,7 @@ module Make = (
               ~chainId,
               ~exn=UndefinedValue,
             )
+          | (None, _) => () //ignore events that aren't registered
```

Comment:
Note, since some events might correctly return None from the eventModLookup, we just ignore those now.